man/fi_domain.3.md

```diff
@@ -492,6 +493,13 @@ shared transmit context.
 The maximum number of endpoints that may be associated with a
 shared receive context.
 
+## Counter Count (cntr_cnt)
+
+The optimal number of completion counters supported by the domain.
+The cq_cnt value may be a fixed value of the maximum number of counters
+supported by the underlying hardware, or may be a dynamic value, based on
+the default attributes of the domain.
```

Comment:
What should the provider return in cases where the hardware doesn't support counters?  In theory there is no optimal number and any number of counters can be supported.